### DISCUSSION TITLE
Navigation in a subview

### DISCUSSION THREAD

**Discussion by FlokkieB** [Clooos/Bubble-Card#802] [Questions about config, custom styles and templates] [answered]
Hi,

nice card! I have pop ups using browser mod. I seems that browser mod is not maintained anymore, so i will replace all my pop ups with this card. But i am facing an issue, using a popup on a subview the card is loosing its navigation. The back arrow doesnt work anymore or returns to the popup. What do i wrong?

**Comment by Clooos** [comment]
Hi, have you tried the latest beta? I've fixed something related to that issue.

**Comment by FlokkieB** [comment]
I’ ll try, when will latest beta be released?

**Comment by FlokkieB** [comment]
Downloaded the beta release. It works, only a little issue left: one has to click twice on the back arrow.

**Reply by Clooos** [reply, marked as answer]
Someone found a way to fix that, I hope this is what you're looking for.

https://github.com/Clooos/Bubble-Card/issues/590#issuecomment-2352996076
